eva-accession-pipeline/src/main/java/uk/ac/ebi/eva/accession/pipeline/batch/tasklets/reportCheck/CoordinatesVcfFactory.java

```diff
@@ -61,7 +61,7 @@
         return variants;
     }
 
-    private Set<String> getIds(String[] fields) {
+    protected Set<String> getIds(String[] fields) {
```

Comment:
I think this is only being user in this class, why change it to protected?

Reply:
compilation failed as I added this method to the parent as protected (like the other methods for parsing each column. I should probably mark this as `@Override`.